Directory.Build.props

```diff
@@ -1,7 +1,7 @@
 <Project>
 
   <PropertyGroup>
-    <Version>1.5.2</Version>
+    <Version>1.5.2-pre</Version>
```

Comment:
This doesn't seem right. Shouldn't this bump the version too?

Reply:
Oh I did that because the no warning for using beta packages was being triggered, somehow , NoWarn flag for individual packages doesn't work so I'll have to disable it globally :/